get-started/what-is-monero/index.md

```diff
@@ -15,8 +15,8 @@ title: "What Is Monero?"
                         <div class="row middle-xs info-block-row private">
                             <div class="col-lg-7 col-md-6 col-sm-8 col-xs-12  why-text">
                                 <h3>Monero is the leading cryptocurrency with a focus on private and censorship-resistant transactions.</h3>
-                                <p>Almost all cryptocurrencies (including Bitcoin and Ethereum) are transparent. They reveal your entire transaction history to the world.</p>
-                                <p>Monero uses unbreakable cryptography to ensure that no one can see how much money you have or where you spend it, even if they know your wallet address.</p>
+                                <p>Most existing cryptocurrencies, including Bitcoin and Ethereum, have transparent blockchains, meaning that transactions are openly verifiable and traceable by anyone in the world. Furthermore, sending and receiving addresses for these transactions may potentially be linkable to a person's real-world identity. </p>
+                                <p>Monero has an obfuscated blockchain. It uses unbreakable cryptography to shield sending and receiving addresses, as well as transacted amounts.</p>
```

Comment:
It is implied when I write "Monero has an obfuscated blockchain. It uses unbreakable cryptography to shield sending and receiving addresses, as well as transacted amounts."